### PR TITLE
Add current weather from Open Weather Map API

### PR DESCRIPTION
**Pull request by denderello**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,15 +1,98 @@
...
+	defer redisCon.Close()
+
+	http.HandleFunc("/", currentWeatherHandler)
+
+	fmt.Println("Starting current weather server")
 	log.Fatal(http.ListenAndServe(":8080", nil))
 }
+
+func currentWeatherHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello world again\n")
```
Comment (luebken) [inline main.go]: can be removed.

Comment (denderello) [inline main.go]: done

File: main.go [hunk 2]
```diff
@@ -1,15 +1,96 @@
...
+	}
+}
+
 func main() {
-	fmt.Println("Starting Helloworld server")
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "Hello world again!!!")
-	})
+	var err error
+	fmt.Println("Establishing connection to Redis")
```
Comment (luebken) [inline main.go]: should use log? no?

Comment (denderello) [inline main.go]: done

**Comment by luebken** [comment]
LGTM

File: main.go [hunk 3]
```diff
@@ -1,15 +1,96 @@
...
-	})
+	var err error
+	log.Println("Establishing connection to Redis")
+	redisCon, err = redis.Dial("tcp", redisAddress())
+	if err != nil {
+		log.Fatalf("Could not connect to Redis with error: %s", err)
+	}
+	defer redisCon.Close()
+
+	http.HandleFunc("/", currentWeatherHandler)
```
Comment (zyndiecate) [inline main.go]: This could have been a job for https://github.com/catalyst-zero/middleware-server

Comment (denderello) [inline main.go]: We wanted to keep it as simple as possible.

**Comment by zyndiecate** [comment]
For a small example app this LGTM.

**Pull request merged**